src/constants/items/forging/pieces.ts

```diff
@@ -72,6 +89,7 @@ export const pieces: { [id: string]: EvoItem } = {
     effects: [],
     crafting: [],
     source: 'Blacksmith',
+    materialFor: ["Crystallized Emerald Blade", "Hand of God", "Tainted Neptune's Eye", "Hades' Artifact", "Tremor", "Paw of Lightning", "Darkforge Plate"]
```

Comment:
```suggestion
    materialFor: ["Crystallized Emerald Blade", "Hand of God", "Tainted Neptune's Eye", "Hades' Artifact", "Tremor", "Darkforge Plate"]
```

not used for paw of lightning (according to ingame shop)